tests/utils/endpoint-test-utils.go

```diff
@@ -0,0 +1,63 @@
+package utils
+
+import (
+	"fmt"
+
+	schema "github.com/devfile/api/pkg/apis/workspaces/v1alpha2"
+)
+
+var Exposures = [...]schema.EndpointExposure{schema.PublicEndpointExposure, schema.InternalEndpointExposure, schema.NoneEndpointExposure}
+
+// GetRandomExposure returns a random exposure value
+func GetRandomExposure() schema.EndpointExposure {
+	return Exposures[GetRandomNumber(len(Exposures))-1]
+}
+
+//var Protocols = [...]schema.EndpointProtocol{schema.HTTPEndpointProtocol, schema.HTTPSEndpointProtocol, schema.WSEndpointProtocol, schema.WSSEndpointProtocol, schema.TCPEndpointProtocol, schema.UDPEndpointProtocol}
```

Comment:
remove comment

Reply:
It is a place holder for when I can run against the latest schema, at which point it will be uncomented.